Examples/Blink/CMakeLists.txt

```diff
@@ -37,19 +37,23 @@ set(STM32CUBEMX_GENERATED_FILES
         Core/Inc/stm32f3xx_it.h
         Core/Inc/gpio.h
         Core/Inc/usart.h
+        Core/Inc/i2c.h
 
         Core/Src/main.cpp
         Core/Src/stm32f3xx_hal_msp.c
         Core/Src/stm32f3xx_it.c
         Core/Src/system_stm32f3xx.c
         Core/Src/gpio.c
         Core/Src/usart.c
+        Core/Src/i2c.c
 
         startup_stm32f303xe.s
         )
 
 set(SOURCES
         # Add user-created source files to this variable
+        ../../Subsurface/Node/Common/peripherals/src

```

Comment:
So we want to be adding specific source files to this variable (even though I know in another set of changes this variable gets globbed), you would want to add `I2CInterface.cpp` to this variable. Also I believe if you look at some of the other branches we are going with uppercase, so it would be `Common/Peripherals/Src`

Reply:
okay, does this mean that the #include in main.cpp would get changed as well from #include "Peripherals/Inc/I2CInterface.h" to #include "I2CInterface.h" ?